src/documents/tokenPrefsServerIntegrationConnection.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Token and Preferences Server Integration Connector
+layout: default
+category: API
+---
+
+## Overview
+
+**Component Name:** `gpii.firstDiscovery.panel.token.prefsServerIntegrationConnection`
+
+**File:** `panels.js`
+
+This connector sends a http request to the first discovery server to save user selected preferences when the token panel becomes visible.
+
+## Using the Token and Preferences Server Integration Connector component
+
+Typically the Token and Preferences Server Integration Connector is integrated into the First Discovery Tool by supplying it in an
+[Auxiliary Schema](http://docs.fluidproject.org/infusion/development/AuxiliarySchemaForPreferencesFramework.html):
```

Comment:
I realize now that this general explanation of what to do with a 'connector' grade could be worded a bit more clearly, here and in the other connector pages: We should say something like "Typically, the X is added as an extra grade to the Y panel specification in the [auxiliary schema](...):"